package.json

```diff
@@ -30,10 +31,13 @@
     "babel-eslint": "^3.1.15",
     "babelify": "^6.1.2",
     "browserify": "^10.2.1",
+    "cerebral": "^0.9.0",
     "eslint": "^0.21.2",
     "eslint-plugin-react": "^2.5.2",
     "express": "^4.12.4",
-    "pre-commit": "^1.0.7"
+    "pre-commit": "^1.0.7",
+    "react": "^0.13.3",
```

Comment:
Don't know Cerebral well enough to know whether it contains internal references to React (hope to get there), vs. mixin, but I think the preference would be that it not be a dependency at all... rather, that it be provided by the requiring module or application.